test/validation/api/crypto/odp_crypto_test_inp.c

```diff
@@ -416,7 +416,7 @@ static void alg_test(odp_crypto_op_t op,
 	/* Create a crypto session */
 	odp_crypto_session_param_init(&ses_params);
 	ses_params.op = op;
-	ses_params.auth_cipher_text = false;
+	ses_params.auth_cipher_text = true;
```

Comment:
why?

Reply:
my understanding of auth_cipher_text = true means do encode then authenticate. and when it is applied together with if (ODP_CRYPTO_OP_ENCODE == param->op) sets the "entry->do_cipher_first"  to true by default and if the param->op == DECODE it sets the !auth_cipher_text. I felt this is more easier for understanding.

Reply:
Theoretically we'd better be testing both combinations. The problem with test changes in your commit is that you change tests rather than expand them. Can you import tests from #379?

Reply:
what do you mean by testing both combinations ?. We are testing both the combinations ENCODE & DECODE. It's the default value we are changing it.

Reply:
both = Encrypt-then-MAC and MAC-cleartext

Reply:
oke I see. I will make the update in the next version.

Reply:
It's really not a problem at this moment, this can be fixed later.